tests/integration_tests/performance/configs/test_network_latency_config_4.14.json

```diff
@@ -72,8 +72,8 @@
                                         "1vcpu_1024mb.json": {
                                             "Avg": {
                                                 "ping": {
-                                                    "delta_percentage": 17.1,
-                                                    "target": 0.225
+                                                    "delta_percentage": 38.1,
```

Comment:
pretty big deltas in this file :/

Reply:
yeah, it seems we got some performance improvement, instead variation came with it.